worker/evCheckerWorker/evCheckerWorker.go

```diff
@@ -0,0 +1,100 @@
+package evCheckerWorker
+
+import (
+	"bytes"
+	"encoding/pem"
+	"fmt"
+	"github.com/mozilla/tls-observatory/logger"
+	"github.com/mozilla/tls-observatory/worker"
+	"io/ioutil"
+	"os/exec"
+	"reflect"
+)
+
+var workerName = "evCheckerWorker"
+var workerDesc = `Provides insights for EV certificates`
+
+var log = logger.GetLogger()
+
+func init() {
+	worker.RegisterWorker(workerName, worker.Info{Runner: new(evWorker), Description: workerDesc})
+}
+
+type evWorker struct{}
+
+func (w evWorker) Run(in worker.Input, res chan worker.Result) {
+	scan, err := in.DBHandle.GetScanByID(in.Scanid)
+	if err != nil {
+		w.error("Could not get scan: %s", err, res)
+		return
+	}
+
+	params := reflect.ValueOf(in.Params)
+	if params.Kind() != reflect.Map {
+		w.error("%s", fmt.Errorf("Invalid parameters passed to evWorker: %s", params), res)
+		return
+	}
+	var oid string
+	for _, k := range params.MapKeys() {
+		if k.Interface() == "oid" {
+			var ok bool
+			oid, ok = params.MapIndex(k).Interface().(string)
+			if !ok {
+				w.error("%s", fmt.Errorf("Could not cast oid to string"), res)
+			}
+		}
+	}
+	fmt.Println("oid:", oid)
```

Comment:
Err, this is just a leftover debugging message I missed when looking over the diff :p